kong/plugins/response-ratelimiting/access.lua

```diff
@@ -75,14 +75,12 @@ function _M.execute(conf)
   -- Load current metric for configured period
   local usage, err = get_usage(conf, identifier, conf.limits, current_timestamp)
   if err then
-    if conf.fault_tolerant then
-      kong.log.err("failed to get usage: ", tostring(err))
-      return
-    else
-      return kong.response.exit(HTTP_INTERNAL_SERVER_ERROR, {
-        message = "An unexpected error occurred"
-      })
+    if not conf.fault_tolerant then
```

Comment:
Why was this `if` changed to a negative? In my opinion it is more difficult to read now.

Reply:
in general, i also prefer to avoid testing the negative, but (at least for me), dealing first with error cases trumps that.  especially if that allows to avoid the `else`

Reply:
early return is the best of all returns